scripts/bundled_installer

```diff
@@ -31,14 +34,25 @@ function exit_if_return_code_is_non_zero() {
     fi
 }
 
+function echo_with_indentation() {
```

Comment:
Why not `export` the equivalent definitions in `python_installer` and simply use them here?